src/components/Search/CompareButton.tsx

```diff
@@ -1,28 +1,19 @@
 import Button from '@mui/material/Button';
-import { style } from 'typestyle';
-
-import { useAppSelector } from '../../hooks/app';
-import { ButtonStyles } from '../../styles';
 
 interface CompareButtonProps {
   label: string;
 }
 
 export default function CompareButton({ label }: CompareButtonProps) {
-  const mode = useAppSelector((state) => state.theme.mode);
-  const btnStyles = ButtonStyles(mode);
-
-  const styles = {
-    button: style({ ...btnStyles.Primary }),
-  };
-
   return (
     <Button
       id='compare-button'
       variant='contained'
-      className={`compare-button ${styles.button}`}
-      sx={{ textTransform: 'none !important' }}
```

Comment:
Initially I wanted to just remove this textTransform that was overriding some other CSS in theme/components (see below).

Note that the text transform override for buttons is already globally set in https://github.com/mozilla/perfcompare/blob/3b2e359b0561746b0f0f94e17d8d6a408ae9001c/src/theme/typography.ts#L11-L13 (as it should be).